index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter">
+    <title>Benjamin Patin's Developer Portfolio</title>
+  </head>
+  <body>
+
+    <!--- Top navigation Bar --->
+    <div class="topnav">
+      <div class="left"></div>
+      <div class="nav-menu">
+        <a href="#news">Portfolio</a>
+        <a href="#about">About</a>
+        <a href="#contact">Contact</a>
+      </div>
+      <div class= "right"></div>
+    </div>
+
+  <!--- Headline Section --->
+    <div class="headline">
+      <div class="headline-text1">Hey There, I am Benjamin Patin.</div>
+      <div class="headline-text2">I am a Full Stack developper.</div>
+      <div class="headline-details"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+don’t hestiate to contact me.</div>
+    </div>
```

Comment:
I would like to highlight that you added comments dividing your sections! It makes it much easier for other programmers like me to read through your code! ✔️ 

- [ ] **[REQUIRED CHANGES]** I would like you to add the background image on the `<div class="headline">` as proposed by the activity. This will make your page pop out even more on the eyes of the recruiters!

- [ ] **[REQUIRED CHANGES]** I would also like you to add the list of the icons as required by the project. This way, whoever visits your page has access to all of your social media.